build-config/resources/Ballerina.toml

```
@@ -5,12 +5,12 @@ name = "hubspot.marketing.transactional"
 version = "@toml.version@"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = [] # TODO: Add keywords
-# icon = "icon.png" # TODO: Add icon
+keywords = ["hubspot","marketing","transactional"]
```

Make sure to build the connector locally after this change to apply the tags.

```suggestion
keywords = ["hubspot","marketing","transactional", "emails"]
```

---

done [39cebf8](https://github.com/ballerina-platform/module-ballerinax-hubspot.marketing.transactional/pull/3/commits/39cebf8975ce224ea0f55eca10f5e74ea4d72ae2)